src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -235,11 +218,29 @@ class PaymentMethodList extends Component {
 
     render() {
         return (
-            <FlatList
-                data={this.createPaymentMethodList()}
-                renderItem={this.renderItem}
-                keyExtractor={item => item.key}
-            />
+            <>
+                <FlatList
+                    bounces={false}
+                    data={this.createPaymentMethodList()}
+                    renderItem={this.renderItem}
+                    keyExtractor={item => item.key}
+                />
+                {
+                    this.props.shouldShowAddPaymentMethodButton
+                    && (
+                        <Button
+                            text={this.props.translate('paymentMethodList.addPaymentMethod')}
+                            icon={Expensicons.CreditCard}
+                            onPress={e => this.props.onPress(e)}
+                            isDisabled={this.props.isLoadingPayments}
+                            style={[styles.mb4, styles.mh4]}
+                            success
+                            shouldShowRightIco
```

Comment:
There's a typo here, `shouldshowRightIcon` and hence `>` isn't showing up in the button.